scopes/dependencies/dependency-resolver/dependency-resolver.docs.md

```diff
@@ -1,235 +1,251 @@
-An extension might provide more dependencies / remove dependencies (uses the hook - see API below).
-The dependency resolver will take this dependencies configuration added by all the hook subscribers and merge them with themselves (2 extensions for the same component might add the same dependency in a different version) - exact strategy to be defined - see open question
-Then it will merge the final result with the result of the previous steps (dependencies configured here are stronger than those from the code analysis)
-#### dependencies added by component config
-The dependency resolver will read special rules configured in the component config and will merge them into the previous results.
-Since these rules added explicitly by the user, they are the strongest ones.
-
-### open issues
-* names of the workspace configuration fields
-* exact policy structure
-* hooks syntax
-* type that the calculation gets as input (the "raw component")
-* how we extend the import / require analysis to enable 3rd party extension to extend it?
-* What is the strategy to merge dependencies added by hooks between themselves (random / load order / alphabetically?)
-* More methods that should be exposed via the API
-* FileDependencies hook name
+#### Setting peer dependencies
+
+Setting a package as a peer dependency ensures the package manager installs only a single version of that package. If that is not possible, if there is no single “agreed upon” version for all components in the workspace then an error will be thrown.
+
+This can be crucial when different components communicate with each other using shared objects that are instantiated by an installed package (the dependency). If different versions of the same package create different object instances then the “means of communication” is broken. There is no single object to address, no single source of truth. This can turn out to be critical when working with modules that are used as “plugins” of another module (for example, Babel), or when working with components that are coordinated in runtime using a shared library (for example, React).
+
+To set a package as a peer dependency, place it under the `peerDependencies` entry, like so:
+
+```json
+{
+  "teambit.bit/dependency-resolver": {
+    "policy": {
+      "dependencies": {},
+      "peerDependencies": {
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1",
+      }
+    }
+}
+```
+
+> Peer dependencies are usually used in the context of a single "hosting code". That could be an application or a single component library. Bit may generate multiple "hosts", one for each environment being used, to run components of different types. That could translate into multiple versions of the same peer dependency, one for each environment. To manage multiple versions of a peer dependency [see here](/docs/faq/multiple-peer-dep-versions).
+
+## CLI reference
+
+### install
+
+To install all dependencies listed in the Dependency Resolver configuration:
```

Comment:
mention that bit install also fetch bit objects (bit import) as part of the install (before the install) and run link after the install.
The import can be disabled by `--skip-import` flag